lib/stage/compressor.rb

```diff
@@ -306,9 +306,8 @@ def copy_tiff_page1(path, destination)
   end
 
   # Set the document name with barcode/image.tif
-  def write_tiff_document_name(path, destination)
-    docname = barcode_file_from_path path
-    cmd = "tiffset -s 269 '#{docname}' #{destination}"
+  def write_tiff_document_name(image_file, destination)
+    cmd = "tiffset -s 269 '#{image_file.barcode_file}' #{destination}"
```

Comment:
I created Issue #90 in response because this affects at least three stage classes. Let's defer the needed cleanup until that issue can be addressed. There is a note that "magic numbers" like this need to get the heave-ho.